tests/lib/rules/id-length.js

```diff
@@ -250,6 +253,7 @@ ruleTester.run("id-length", rule, {
         { code: "var [,i,a] = arr;", languageOptions: { ecmaVersion: 6 }, errors: [tooShortError, tooShortError] },
         { code: "function foo([a]) {}", languageOptions: { ecmaVersion: 6 }, errors: [tooShortError] },
         { code: "import x from 'module';", languageOptions: { ecmaVersion: 6 }, errors: [tooShortError] },
+        { code: "import { x as z } from 'module';", languageOptions: { ecmaVersion: 6 }, errors: [tooShortError] },
```

Comment:
```suggestion
        { code: "import { x as z } from 'module';", languageOptions: { ecmaVersion: 6 }, errors: [{ ...tooShortError, column: 15 }] },
        { code: "import { foo as z } from 'module';", languageOptions: { ecmaVersion: 6 }, errors: [{ ...tooShortError, column: 17 }] },
        { code: "import { 'foo' as z } from 'module';", languageOptions: { ecmaVersion: 2022 }, errors: [{ ...tooShortError, column: 19 }] },
```

It would be good to check the `column` to verify that the right identifier is reported, and also add a test where the imported name is of a valid length and a test with string literal.